CONTRIBUTORS.yaml

```diff
@@ -212,6 +212,9 @@ tnabtaf:
 torhou:
     name: Torsten Houwaart
 
+torstenseemann:
```

Comment:
this should be equal to the github username of the contributor, which I believe is `tseemann` right?

Reply:
Yes, thanks!

Reply:
I just used the online editor for that change, hope that's ok.